tests/graphql_directive_decl/directive_decl.graphql

```diff
@@ -13,4 +13,4 @@ directive @a(as: String) on QUERY
 
 
 
-directive @a(as: String = 1) on QUERY
+directive @a(as: String = 1) repeatable on QUERY
```

Comment:
can you add new test case instead of modifying existing one?

Reply:
Sure thing.